sdk/src/types.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
 
 pub use diem_types::*;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Major requirement is that we cannot clone private keys or these accounts.

Reply:
yea, you are right, sorry I am still doing cleaning stuff, I will make sure this clone will be removed which it was used for trial purpose and I forgot to remove 😿

Reply:
same of the clone below